db/migrations/20191120190404_user.js

```diff
@@ -0,0 +1,20 @@
+
+exports.up = function(knex) {
+  return Promise.all([
+    knex.schema.createTable('users', function (table) {
+      table.increments('id').primary();
+      table.string('email');
+      table.string('password');
+      table.string('apiKey');
+      table.string('favoriteLocation');
```

Comment:
users can have multiple favorite locations, might make things easier down the line to create a favorites table with a location and userId

Reply:
I see. Creating a separate table for the favorites seems like a much better design.